db.tf

```
@@ -5,7 +5,7 @@ module "tt-database" {
   location              = var.location
   env                   = var.env
   postgresql_user       = var.db_postgresql_user
-  database_name         = "${var.product_name_short}-${var.env}"
+  database_name         = "db-${var.env}"
```

i wouldn't put the env in here you don't need it
```suggestion
  database_name         = "${var.product}"
```